docs/catalog.md

```diff
@@ -64,7 +64,6 @@ search_parameters = catalog.construct_search_parameters(
     start_date="2022-06-01",
     end_date="2022-12-31",
     max_cloudcover=20,
-    sortby="cloudCoverage",
```

Comment:
@dustgalactic 
FYI: Removing `sortby` as [it is being deprecated.](https://github.com/up42/up42-py/blob/58f8e27118d2e1d068b4a4fa12c379cecfcffca4/up42/catalog.py#L238).

Reply:
@up42/documentation for future reference: whenever you come across anything like this, can you please create a ticket for us as well so we can also remove it from the code altogether.

Reply:
created a ticket for us [here](https://up42.atlassian.net/browse/AT-3863)